Gemfile

```diff
@@ -17,7 +17,9 @@ group :development, :test do
   gem 'coveralls', require: false
   gem 'pry-rails'
   gem 'pry-byebug'
+  gem 'rspec'
```

Comment:
Why do you add this gem? There's already a gem called `rspec-rails` in the Gemfile

Reply:
I will remove it